airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -1,91 +1,85 @@
-connector_image: airbyte/source-hubspot:dev
 acceptance_tests:
-  spec:
+  basic_read:
     tests:
-      - spec_path: "source_hubspot/spec.yaml"
-        backward_compatibility_tests_config:
-          # API Key authentication is deleted from authorization choices so it is not backward compatible with previous spec
-          disable_for_version: "0.2.3"
+    - config_path: secrets/config.json
+      empty_streams:
+      - name: companies
+        bypass_reason: hs_time_in_opportunity field changes after every read
+      - name: contacts
+        bypass_reason: data is changing after every read
```

Comment:
At least these 2 streams should have consistent data - what's changing?

Reply:
I have checked the data for these two streams, removed `contacts` stream as it indeed has consistent data, last test run failed due to some issues from API side (maybe some kind of a rate limit), will re-run them later or investigate what may cause this failed run